server/seeder.js

```diff
@@ -0,0 +1,209 @@
+Dr. Strangelove
+The Lives of Others
+Grave of the Fireflies
+Paths of Glory
+Django Unchained
+The Shining
+WALL-E
+American Beauty
+Princess Mononoke
+The Dark Knight Rises
+Blade Runner 2049
+Oldboy
+Witness For the Prosecution
+Aliens
+Once Upon a Time in America
+Das Boot
+Dangal
+Citizen Kane
+Vertigo
+North By Northwest
+Star Wars: Episode VI - Return of the Jedi
+Braveheart
+Reservoir Dogs
+M
+Requiem for a Dream
+Your Name
+Like Stars on Earth
+Amelie
+A Clockwork Orange
+Lawrence of Arabia
+Amadeus
+Double Indemnity
+Eternal Sunshine of the Spotless Mind
+Taxi Driver
+To Kill A Mockingbird
+Full Metal Jacket
+Singin In The Rain
+2001: A Space Odyssey
+Toy Story
+3 Idiots
+The Sting
+Toy Story 3
+Inglourious Basterds
+The Bicycle Thief
+The Kid`;
+
+var movieNamesArray = movieNames.split('\n');
+//console.log(movieNamesArray);
+var tagsArray = ['comedy', 'horror', 'drama', 'holiday special', 'action', 'adventure', 'scifi', 'fantasy', 'thriller', 'animated'];
+var mpaaRatings = ['G', 'PG', 'PG-13', 'R', 'NC-17', 'Unrated'];
+
+db.connectAsync()
+  .then(() => db.queryAsync(`DROP DATABASE IF EXISTS ${database}`))
+  .then(() => db.queryAsync(`CREATE DATABASE ${database}`))
+  .then(() => db.queryAsync(`USE ${database}`))
+  .then(() => db.queryAsync(`
+    CREATE TABLE Features (
+      id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+      title VARCHAR (50),
+      category_1 VARCHAR(30),
+      category_2 VARCHAR(30),
+      release_date INT,
+      mpaa_rating VARCHAR(15),
+      length INT,
+      star_rating INT,
+      star_rating_count INT,
+      rt_rating INT,
+      description VARCHAR(700),
+      hd_rent INT,
+      sd_rent INT,
+      hs_cost INT,
+      sd_cost INT,
+      movie_shot_url VARCHAR(30),
+      movie_cover_url VARCHAR(30)
+    )`))
+  .then(() => db.queryAsync(`
+    CREATE TABLE Wishlist (
+      id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+      film_name VARCHAR(20)
+  )`))
+  .then(() => {
+    for(var x = 0; x < 100; x++) {
+      db.queryAsync(`INSERT INTO Features (
+        title,
+        category_1,
+        category_2,
+        release_date,
+        mpaa_rating,
+        length,
+        star_rating,
+        star_rating_count,
+        rt_rating,
+        description,
+        hd_rent,
+        sd_rent,
+        hs_cost,
+        sd_cost,
+        movie_shot_url,
+        movie_cover_url
+      ) VALUES (
+        '${movieNamesArray[Math.floor(Math.random()*movieNamesArray.length) -1]}',
+        '${tagsArray[Math.floor(Math.random()*tagsArray.length) -1]}',
+        '${tagsArray[Math.floor(Math.random()*tagsArray.length) -1]}',
+        ${1930 + Math.floor(Math.random()* 90)},
+        '${mpaaRatings[Math.floor(Math.random()*mpaaRatings.length) -1]}',
+        ${Math.floor(Math.random()*200)},
+        ${Math.random()*5},
+        ${Math.floor(Math.random()*5000)},
+        ${Math.floor(Math.random()*100)},
+        '${lorem.generateParagraphs(2)}',
+        ${Math.floor(Math.random()*2000)/100},
```

Comment:
You should always store prices as "number of cents" in your database due to how JavaScript handles numbers. For example, store `1999` for `$19.99`.
```
1.01 + 2.01 === 3.02 // false
```